docs/standard/garbage-collection/fundamentals.md

```diff
@@ -210,55 +198,79 @@ The following are threading and performance considerations for server garbage co
+![Background server garbage collection](./media/fundamentals/background-server-garbage-collection.png)
 
-The following illustration shows background garbage collection performed on a separate dedicated thread on a workstation:
+## Concurrent garbage collection
 
-![Diagram that shows background workstation garbage collection.](./media/fundamentals/background-workstation-garbage-collection.png "Diagram that shows background workstation garbage collection.")
+> [!TIP]
+> This section applies to:
+>
+> - .NET Framework 3.5 and earlier for workstation garbage collection
+> - .NET Framework 4 and earlier for server garbage collection
+>
+> Concurrent garbage is replaced by [background garbage collection](#background-garbage-collection) in later versions.
 
-## Background server garbage collection
+In workstation or server garbage collection, you can enable concurrent garbage collection, which enables threads to run concurrently with a dedicated thread that performs the garbage collection for most of the duration of the collection. This option affects only garbage collections in generation 2; generations 0 and 1 are always non-concurrent because they finish very fast.
 
-Starting with the .NET Framework 4.5, background server garbage collection is the default mode for server garbage collection. To choose this mode, set the `enabled` attribute of the [\<gcServer> element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) to `true` in the runtime configuration schema. This mode functions similarly to background workstation garbage collection, described in the previous section, but there are a few differences. Background workstation garbage collection uses one dedicated background garbage collection thread, whereas background server garbage collection uses multiple threads, typically a dedicated thread for each logical processor. Unlike the workstation background garbage collection thread, these threads do not time out.
+Concurrent garbage collection enables interactive applications to be more responsive by minimizing pauses for a collection. Managed threads can continue to run most of the time while the concurrent garbage collection thread is running. This results in shorter pauses while a garbage collection is occurring.
 
-The following illustration shows background garbage collection performed on a separate dedicated thread on a server:
+Concurrent garbage collection is performed on a dedicated thread. By default, the CLR runs workstation garbage collection with concurrent garbage collection enabled. This is true for single-processor and multi-processor computers.
+
+The following illustration shows concurrent garbage collection performed on a separate dedicated thread.
+
+![Concurrent Garbage Collection Threads](./media/gc-concurrent.png)
+
+## Configure garbage collection
```

Comment:
> Configure garbage collection [](start = 2, length = 29)

should this just point to the page that you are working on, @gewarren?

Reply:
Yes, good idea. I added the link, which will be broken until that PR is merged.